README.md

```diff
@@ -1,2 +1,134 @@
-# vonage-spring-boot-starter-java
-Spring Boot Starter for Vonage
+# vonage Spring Boot Starter
```

Comment:
The "v" in vonage should be capitalized. Also add the world "Java" to the ends so that it matches the name of the project.